scripts/src/java/org/oppia/android/scripts/docs/KDocValidityCheck.kt

```diff
@@ -72,24 +74,40 @@ fun main(vararg args: String) {
   )
 
   // A list of all the files to be exempted for this check.
-  val kdocExemptionList = loadKdocExemptionsProto(kdocExemptiontextProto).getExemptedFilePathList()
+  val kdocExemptionList = loadKdocExemptionsProto(pathToProtoBinary)
+    .getExemptedFilePathList()
```

Comment:
```suggestion
  val kdocExemptionList =
    loadKdocExemptionsProto(pathToProtoBinary).getExemptedFilePathList()
```

Wrapping should be at the higher syntactical element (in this case the ``=``).

Reply:
Done.